app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/returns/address/acquisition_price.scala.html

```diff
@@ -43,24 +44,32 @@
     form: Form[BigDecimal],
     backLink: Call,
     isATrust: Boolean,
-    representativeType: Option[RepresentativeType]
+    representativeType: Option[RepresentativeType],
+    dateOfDeath: Option[DateOfDeath]
 )(implicit request: RequestWithSessionData[_], messages:Messages, appConfig: ViewConfig)
 
 @key = @{ "multipleDisposalsAcquisitionPrice" }
 @isAgent = @{ request.userType.contains(UserType.Agent) }
 @userKey = @{
     representativeType match {
       case Some(PersonalRepresentative) => ".personalRep"
-      case Some(PersonalRepresentativeInPeriodOfAdmin) => ".personalRep"
+      case Some(PersonalRepresentativeInPeriodOfAdmin) if(!isAgent)=>  ".personalRepInPeriodOfAdmin"
```

Comment:
did you update the summary template? The title's need updating there as well

Reply:
🤦  added a "Do not merge" label